packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx

```diff
@@ -349,6 +361,16 @@ async function handlePaymentIntent({
       return onSuccess();
     }
     case 'requires_payment_method': {
+      // If this happens when a customer is trying to subscribe to their first
+      // product, detach the payment method.  The user must enter a working
+      // card to move on; we'll get a working payment method from that.
+      // This does not affect functionality, thus not blocking.
+      if (
+        (!customer || (customer && customer.subscriptions.length === 0)) &&
+        paymentMethodId
+      ) {
+        apiDetachFailedPaymentMethod({ paymentMethodId });
```

Comment:
I think there might be a *slim* chance here that we could end up leaving a failed payment method in place - i.e. since we don't have retries, a network error at this point would do it. 

But it's so slim a chance that it's probably just a thing to keep in mind if a customer service issue comes up later.